collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -61,6 +61,7 @@
 
 import javax.inject.Inject;
 
+import static android.app.Activity.RESULT_CANCELED;
```

Comment:
The test is failing because of this unused import

Reply:
oh thanks @shobhitagarwal1612 i see now i was worried. That was when i tried checking other conditions in case the user cancelled etc forgot to remove that. will make a commit. Thanks